projects/cloudflare/cfssl/docker/linux/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+copy _output/gitdependencies/cfssl_trust /etc/cfssl
```

Comment:
COPY

Reply:
Also, do you need the entire cloned repo in that container image?  Can we limit that?

Reply:
Thanks! and yea I think we do? I saw on their docker file they actually clone the whole repo.
https://github.com/cloudflare/cfssl/blob/master/Dockerfile#L6
I searched their code it did look like they were referencing files from there but probably not all of it.

Reply:
Man that feels so lazy